kernel/packages/scene-system/stateful-scene/BuilderStatefulActor.ts

```diff
@@ -0,0 +1,51 @@
+﻿import { CatalystClient } from 'dcl-catalyst-client'
+import { EntityType } from 'dcl-catalyst-commons'
+import { SceneStateDefinition } from './SceneStateDefinition'
+import { ILand } from 'shared/types'
+import { SceneStateStorageController } from 'shared/apis/SceneStateStorageController/SceneStateStorageController'
+import { deserializeSceneState } from './SceneStateDefinitionSerializer'
+
+declare var window: any
+
+export class BuilderStatefulActor {
```

Comment:
Why are we calling this a `StatefulActor` if it doesn't extend `StatefulActor`? It probably makes sense depending on how we expect it to evolve, but I wanted to understand your plans for it

Reply:
The idea behind this actor is to handle the state of the scene based on the server state.

Right now since it only asks the builder server, it is more descriptive that way, however, in the future when the decentraliced server system comes in, it should be renamed to something like `ServerStatefulActor` and it should manage all the interaction with the server. So for me, it is an actor that handles the communication with the server. 

I don't know if this makes sense to you, please let me know!